unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -0,0 +1,106 @@
+using DCL.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace DCL.EquippedEmotes
+{
+    /// <summary>
+    /// Plugin feature that initialize the Equipped Emotes feature.
+    /// </summary>
+    public class EquippedEmotesPlugin : IPlugin
+    {
+        internal const string PLAYER_PREFS_EQUIPPED_EMOTES_KEY = "EquippedNFTEmotes";
+
+        internal DataStore_EmotesCustomization emotesCustomizationDataStore => DataStore.i.emotesCustomization;
+
+        public EquippedEmotesPlugin()
+        {
+            LoadDefaultEquippedEmotes();
+
+            emotesCustomizationDataStore.isEmotesCustomizationInitialized.OnChange += OnEmotesCustomizationInitialized;
```

Comment:
Cannot we use the featureFlag instead of this boolean? Preparing the data should be a independent process unrelated to the customization of emotes.

If featureFlag is on, you set the emotes from PlayerPrefs and listen to changes in th equipped ones to save them
If not, you just put the default ones

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2078/commits/10e2e1bfa9405fae91638856d22d2e334702087e.